plugin-hrm-form/src/states/case/types.ts

```diff
@@ -171,6 +171,7 @@ export type CaseDetails = {
   referrals: t.ReferralEntry[];
   notes: NoteActivity[];
   documents: t.DocumentEntry[];
+  caseSummaries?: t.CaseSummaryEntry[];
```

Comment:
This should not be an array or optional, cases will always have 1 case summary,
```suggestion
  caseSummary: t.CaseSummaryEntry;
```

Reply:
@stephenhand I removed it. Turns out that the declaration is totally useless here!